src/main/java/org/breedinginsight/brapps/importer/services/processors/experiment/ExperimentWorkflowNavigator.java

```diff
@@ -0,0 +1,72 @@
+package org.breedinginsight.brapps.importer.services.processors.experiment;
+
+import io.micronaut.context.annotation.Primary;
+import org.breedinginsight.brapps.importer.model.imports.ImportServiceContext;
+import org.breedinginsight.brapps.importer.model.workflow.ImportWorkflow;
+import org.breedinginsight.brapps.importer.model.workflow.ExperimentWorkflow;
+import org.breedinginsight.brapps.importer.model.workflow.ImportWorkflowResult;
+
+import javax.inject.Singleton;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Primary
+@Singleton
+public class ExperimentWorkflowNavigator implements ExperimentWorkflow {
+    private final List<ExperimentWorkflow> workflows;
```

Comment:
It seems like this is the class that orchestrates the running of the workflows, having a high-level explanation in a comment might help other developers understand the architecture.

Reply:
I've added overview comments in the WorkflowNavigator to be merged in with [BI-2134](https://github.com/Breeding-Insight/bi-api/pull/359/files#diff-83f2deb13f2ea62ba49ed53c10bf4bc30ccf63cf6105a130a9c9056597f2a5c3).